Sources/FeaturevisorSDK/Instance+Refresh.swift

```diff
@@ -21,7 +21,8 @@ extension FeaturevisorInstance {
 
         try? fetchDatafileContent(
             from: datafileUrl,
-            handleDatafileFetch: handleDatafileFetch) { [weak self] result in
+            handleDatafileFetch: handleDatafileFetch
```

Comment:
Please, double check that these format changes follow the current swift-format config

Reply:
Updated the branch from the main, this change disappeared on its own + swiftformating shows no errors